variables.tf

```diff
@@ -190,38 +196,12 @@ variable "packet_mirror_network_tag" {
   description = "the packet mirror policy tag for mirrored resources"
 }
 
-variable "enrichment_bucket_name" {
-  description = "(optional) the gcs bucket name"
-  type        = string
-  default     = ""
-}
-
 variable "sensor_service_account_email" {
   description = "When enrichment is configured, this must be set to a service account which has the required permissions"
```

Comment:
I see that there's still a mention of enrichment here, should that be removed as well? Or modified?